storybook/stories/ServerItem.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions, react/prop-types */
+import React from 'react';
+import { SafeAreaView, FlatList } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+
+import ServerItem from '../../app/presentation/ServerItem';
+import { ThemeContext } from '../../app/theme';
+
+const stories = storiesOf('ServerItem', module);
+
+const themes = {
+	light: 'light',
+	dark: 'dark',
+	black: 'black'
+};
+
+const item = {
+	name: 'https://open.rocket.chat/',
+	id: 'some.user',
+	iconURL: 'https://open.rocket.chat/images/logo/android-chrome-512x512.png'
+};
+
+const item2 = {
+	name: 'https://superlongservername.rocket.chat/',
+	id: 'superlongusername.tologintoasuperlongservername'
+};
+
+stories.add('unselected server', () => (
```

Comment:
You don't have to add one component per story.
There's no rule, but this way is not very useful.

See `Thread Messages.Item`.
`content` story is to test the different aspects of the component.
Then we have `badges` to test different badge types and `themes`.